credentials/apps/credentials/views.py

```diff
@@ -149,7 +149,12 @@ def get_context_data(self, **kwargs):
                 display_name='Absolutely Fake University',
                 certificate_logo_image_url='https://placehold.it/204x204'
             )],
-            hours_of_effort=None
+            hours_of_effort=None,
+            courses=[
```

Comment:
is it compulsory to make this change? as it's core change? I guess we can leave it as it was.

Reply:
Actually credential service gives an example view to check credential design with example data instead of completing the whole flow **syncing courses with discovery -> create program -> create program certificate config -> add signatories -> Add user credential**  to see the certificate. As we have integrated our theme, That's why I also updated this view according to our theme and requirement, So that this view still works properly.